src/frontend/packages/cloud-foundry/src/features/cf/tabs/cf-organizations/cf-organization-spaces/tabs/cloud-foundry-space-users/cloud-foundry-space-users.component.scss

```diff
@@ -0,0 +1,11 @@
+.sub-nav-buttons {
```

Comment:
Is there any way we can put this styling on the sub-nav component, so we don't need to repeat it?

Reply:
I'd put them in the individual files due to the different style context (`ngcontent-vhq-cXXX`) of the parent `page-header-sub-nav__container` and buttons that stops the styles being applied. I had a play, could take the same approach as the theme file `dashboard-base.component.theme.scss` that's already edited in this PR?